index.js

```diff
@@ -0,0 +1,217 @@
+  Tecnologies: ['HTML', 'bootstrap', 'Ruby'],
+  Link: 'see my project',
+},
+{
+  Name: 'Profesional Art Printing Data',
+  Description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+  Tecnologies: ['HTML', 'bootstrap', 'Ruby'],
+  Link: 'see my project',
+},
+{
+  Name: 'Profesional Art Printing Data',
+  Description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+  Tecnologies: ['HTML', 'bootstrap', 'Ruby'],
+  Link: 'see my project',
+},
+{
+  Name: 'Profesional Art Printing Data',
+  Description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+  Tecnologies: ['HTML', 'bootstrap', 'Ruby'],
+  Link: 'see my project',
+},
+{
+  Name: 'Profesional Art Printing Data',
+  Description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+  Tecnologies: ['HTML', 'bootstrap', 'Ruby'],
+  Link: 'see my project',
+}];
+
+//  Project Section Card creator
+const cardContainer = document.querySelector('#section-02');
+for (let i = 0; i < ProjectsMobile.length; i++) {
+  const projectM = `<div class="card-2">
+  <h2 class="prof-art">${ProjectsMobile[i].Name}</h2>
+  <p class="p22">${ProjectsMobile[i].Description}</p>
+  <ul class="skils">
+    <li class="button">${ProjectsMobile[i].Tecnologies[0]}</li>
+    <li class="skil button">${ProjectsMobile[i].Tecnologies[1]}</li>
+    <li class="skil button">${ProjectsMobile[i].Tecnologies[2]}</li>
+  </ul>
+  </div>
+  <button class ="see-project-2" id="seeproject${[i]}" onclick="popupp()">${ProjectsMobile[i].Link}</button>
+  `;
+  cardContainer.innerHTML += projectM;
+}
+
+// Get the popup mobile
+const modal = document.getElementById('myModal');
+modal.innerHTML = `<div class="modal-content">
+<div class="modalHead-close">
+<h2 class = "modalHeader" id="modalHeader">Multi Post Stories</h2>
+ <span class="close">&times;</span>
+ </div>
+ <div>
+  <ul id="modalTech" class ="modalTech">
+      <li>HTML</li>
+      <li>Bootstrap</li>
+      <li>Ruby on rails</li>
+  </ul>
+  <img src="/assets/Portfolio.png" alt="popup image">
+  <p class = "modal-pm" >Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent</p>
+  <div class = "live-source">
+  <a href="https://www.google.com/">see live <img src="/assets/Icon.png" alt="google"></a>
+  <a href="https://github.com/">see source <img src="/assets/Vector.png" alt=""></a>
+  </div>
+</div>
+</div>`;
```

Comment:
- [ ] Kindly make sure the popup modal for each project is populated with the data from the projects array and not manually like the above. ✔